person/src/homeworks/person/core/model/Singer.java

```diff
@@ -1,21 +1,32 @@
 package homeworks.person.core.model;
 
+import homeworks.person.core.enums.Profession;
+import homeworks.person.core.enums.SingerType;
+import homeworks.person.core.exceptions.InvalidNameException;
 import homeworks.person.core.service.SingerAction;
+import homeworks.person.core.service.impl.PersonActionFactory;
 
 public class Singer extends BasePerson implements SingerAction {
+  private SingerType designation;
   private String bandName;
 
-  public Singer (String name, String designation, String bandName)
+  private boolean full;
```

Comment:
@tatevikb what is the default value of this fields ?

Reply:
Default values are false for `full`, `learned`, `walked`, that's why I didn't assign the values, but I've updated the code for these fields, now values are assigned  `private boolean full = false` ets..